BRIDGES.md

```diff
@@ -0,0 +1,48 @@
+# Using BRIDGES dependency (`git subtree`).
```

Comment:
I wouldn't stylize this in all caps - it's not an acronym. I'd write this as: "Using the Parity Bridges Common Dependency", and use either "Bridges" or `parity-bridges-common` in the rest of the doc.